{"id": "5340", "title": "Bug 5340 - Cancelling add exception breakpoint has no effect", "description": "", "OB": "When trying to add an exception breakpoint, the progress dialog closes immediately, and the exception list comes up even if the cancellation request was made. It does not honor the cancellation request. The cancel button should either disable or the cancellation request should be honored by the system. This can be achieved by passing 'false' for the 'cancelable' parameter when calling 'ProgressMonitorDialog.run'.", "EB": "", "SR": "To reproduce the issue: 1) Go to the breakpoints pane in the debug perspective. 2) Click the 'J!' button to add an exception breakpoint. 3) In the progress dialog, click cancel."}